homeassistant/components/fyta/__init__.py

```diff
@@ -22,11 +30,16 @@
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up the Fyta integration."""
+    tz: str = hass.config.time_zone
 
     username = entry.data[CONF_USERNAME]
     password = entry.data[CONF_PASSWORD]
+    access_token: str = entry.data.get(CONF_ACCESS_TOKEN, "")
```

Comment:
```suggestion
    access_token: str = entry.data[CONF_ACCESS_TOKEN]
```
You can expect an access token to be present